### PR TITLE
segmentbutton

### PR DESCRIPTION
**Pull request by Amitrawat21**
segment button

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1 +1,2 @@
-import './components/buttons/index' // import all buttons component here to the entry point for the npm package
\ No newline at end of file
+import './components/buttons/index' // import all buttons component here to the entry point for the npm package
```
Comment (tabassum2507) [inline src/index.ts]: remove buttons from index.ts

File: src/interface/SegmentedButtonsInterface/SegmentedButtonsInterface.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+ 
+ export interface SegmentedButtonsProps {
+  segments: { value: string; label: string }[];
```
Comment (tabassum2507) [inline src/interface/SegmentedButtonsInterface/SegmentedButtonsInterface.ts:3]: make fields mandatory

File: src/components/SegmentedButtons/SegmentedButtons.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import { View } from 'react-native';
+import { SegmentedButtons } from 'react-native-paper';
+import { SegmentedButtonsProps } from '../../interface/SegmentedButtonsInterface/SegmentedButtonsInterface';
+
```
Comment (tabassum2507) [inline src/components/SegmentedButtons/SegmentedButtons.tsx:5]: remove blank spaces

**Review by tabassum2507 — CHANGES_REQUESTED** [review]

**Review by tabassum2507 — APPROVED** [review]

File: src/interface/SegmentedButtonsInterface/SegmentedButtonsInterface.ts (hunk 2 above)
Comment (Amitrawat21) [reply src/interface/SegmentedButtonsInterface/SegmentedButtonsInterface.ts:3]: resolved

**Review by rohan-arya-w — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (Amitrawat21) [reply src/index.ts]: resolved

File: src/components/SegmentedButtons/SegmentedButtons.tsx (hunk 3 above)
Comment (Amitrawat21) [reply src/components/SegmentedButtons/SegmentedButtons.tsx:5]: resolved

**Pull request merged**